tests/scenarios/github_test_12/cmd3

```diff
@@ -0,0 +1,7 @@
+source: ubuntu-1-deployment
+cmd: wget --tries=1 142.250.193.46
```

Comment:
what is the difference between cmd1 and cmd3?

Reply:
We have a policy for allowing tcp for curl, this tries to access tcp using wget which should be blocked due to our default posture